src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+    title: ESLintバージョン
+    latest: 最新バージョン
+    upcoming: 次のバージョン
+    development: 開発中バージョン
+    dateline: <span class="text-dark">VERSION</span>（DATE）
+
+  explainer:
+    image:
+      src: eslint-features-img.png
+      alt: "ESLintが動作しているVisual Studioのコードウィンドウのスクリーンショット：IDE内のESLintエラーに下線を引いています。"
+    title: The pluggable linting utility for JavaScript and JSX
+    description: >
+      ESLintは、あなたのJavaScriptコードの問題を発見し、修正することを支援するオープンソースプロジェクトです。
+      ブラウザのためか、サーバのためか、フレームワークを使用しているかに関わらず、ESLintはあなたのコードが
+      良い状態になるように支援します。
+    features:
+      - title: 間違いを探す
+        description: >
+          ESLintは、コードを静的に解析し、問題を迅速に発見します。ESLintは ほとんどのテキストエディタに組み込まれており、
+          ESLintをCIパイプライン内で実行することもできます。
+        learn_more:
+          link: getStarted
+          text: >
+            <span class="visually-hidden">ESLintで間違いを探す方法について</span>もっと知りたい
+      - title: 自動的に修正する
+        description: >
+          ESLintが発見した問題のほとんどは、自動的に修正することができます。
+          ESLintは構文を意識して修正を行うため、従来の置換ベースのコード修正アルゴリズムのように新しいエラーを生み出すことはありません。
+        learn_more:
+          link: fixProblems
+          text: >
+            <span class="visually-hidden">ESLintの自動修正について</span>もっと知りたい
+      - title: Configure everything
+        description: >
+          コードの前処理、カスタムパーサーの使用、ESLintの組み込みルールと一緒に動作する独自のルールを作成することができます。
+          プロジェクトで必要な形に合わせてESLintをカスタマイズすることができます。
+        learn_more:
+          link: configuring
+          text: >
+            <span class="visually-hidden">ESLintのカスタマイズについて</span>もっと知りたい
+  blog:
+    view_all_posts: 全てのブログポストを見る
+
+  stats:
+    title: ようこそ、コミュニティへ
+    description: >
+      ESLintは、npmでのダウンロード数でNo.1のJavaScriptリンターです（DOWNLOAD_COUNT以上）。
+      Microsoft、Airbnb、Netflix, Facebookなどの企業で使用されています。
+    dependents: ディペンデンツ
+    weekly_downloads: 週間ダウンロード数
+    stars: スター
+
+  sponsors:
+    title: 素晴らしい人々によって支えられている
+    description: >
+      一年でSPONSOR_COUNTのスポンサー様からANNUAL_DONATIONS の寄付をいただいています。
+    view_all_sponsors: スポンサーの皆様を見る
+
+  donations:
+    title: 最近の寄付
+    view_all_donations: 全ての寄付を見る
+
+#------------------------------------------------------------------------------
+# Blog Page
+#------------------------------------------------------------------------------
+
+blog_page:
+  title: ニュース・最新情報
+  description: ESLintの最新ニュース、ケーススタディ、チュートリアル、リソースなどをご紹介します。
+  search:
+    title: 検索
+    hint: キーワードを入力するとリアルタイムに検索結果が表示されます。
+    clear: 検索の解除
+  categories:
+    title: カテゴリー一覧
+    everything: 全て
+  pagination:
+    title: ページ
+    first_page: 最初
+    next_page: 次
+    previous_page: 前
+    last_page: 最後
+    status: TOTAL_PAGE_COUNT ページ中の PAGE_NUMBER ページ目
+
+#------------------------------------------------------------------------------
+# Team Page
+#------------------------------------------------------------------------------
+
+team_page:
+  title: チーム紹介
+  description: >
+    ESLintを構築し、維持管理しているチームメンバーです
+  tsc:
+    title: 技術運営委員会
+    description: >
+      リリースを管理し、機能要求をレビューし、ESLintが適切に維持されるように定期的に
+      ミーティングを行う人たちです。
```

```suggestion
      ミーティングを行う人々
```
I thought the end of the sentence should be unified